2-multitenant/envs/development/outputs.tf

```
@@ -34,6 +29,11 @@ output "env" {
   value       = local.env
 }
 
+output "cluster_regions" {
+  description = "Regions where Cluster were deployed"
```

nit: suggestion only to avoid the present/past tense

```suggestion
  description = "Regions with clusters"
```